gradle/lite.gradle

```diff
@@ -0,0 +1,58 @@
+/**
+ * This gradle file add task `liteLibJarAll` which combines lite jar library files into one.
+ * It is helpful if project contains multiple application with different set of dependencies.
+ *
+ * For using this task is required build lite jars of application before and then it is possible
```

Comment:
Not clear on what this comment is trying to explain for the `liteLibJarAll` task. If I understand correctly, this might be more clear:

```
For `liteLibJarAll` to be used, lite jars of each application must be built first. Then this task will provide a big library file that contains each application. This will reduce the size of data when packaging and deploying.
```